e2e/framework/utils/k8s_helper.go

```diff
@@ -0,0 +1,253 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/jmoiron/jsonq"
+	"html/template"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type K8sHelper struct {
+}
+
+func CreatK8sHelper() *K8sHelper {
+	return &K8sHelper{}
+}
+
+func (k8sh *K8sHelper) getMonIp(mon string) (string, error) {
```

Comment:
Again, all this functions would be easier to use if using the Kubernetes client binding.